### DISCUSSION TITLE
Why checkbox not working in Canvas

### DISCUSSION THREAD

**Discussion by khanhleemtp** [GrapesJS/grapesjs#4415] [Q&A]
When i click checkbox or add checked properties checked, checkbox not display checked. Please help me

**Comment by ronaldohoch** [comment]
Can you provide some code to check?

**Reply by bfullstack** [reply]
I am facing same issue in my custom design of grapes-js

**Reply by mosh-tudor** [reply]
@bfullstack That's nice, but can you

> provide some code to check?

?
